src/features/pages/partners/deriv-prime/meet-us-there/data.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import * as styles from './meet-us-there.module.scss'
+import { ExpoCardTypes } from './types'
+
+export const expo_cards: ExpoCardTypes[] = [
+    {
+        title: '_t_iFX EXPO Cyprus_t_',
+        date: '_t_19-21 September 2023_t_',
+        image: (
+            <StaticImage
+                src="../../../../../images/common/deriv-prime/dubai_expo.png"
+                alt="Dubai Expo"
```

Comment:
Can you please add translation for the `alt` eg:` alt={localize('_t_Dubai Expo_t_')}`